ansible_collections/arista/avd/molecule/evpn_underlay_ebgp_overlay_ebgp/intended/configs/DC1-BL1A.cfg

```diff
@@ -105,31 +105,31 @@ interface Ethernet10.200
    ip address 10.10.21.10/24
 !
 interface Ethernet41
-   description P2P_LINK_TO_DC1-SPINE1_Ethernet6
+   description CUSTOM_P2P_LINK_TO_DC1-SPINE1_Ethernet6 
```

Comment:
Seems template adds extra whitespace at EOL

Reply:
True, although that is just a basic example of a customer provided template. It adds a trailing space when switch.struct_cfg.tags is empty. I will elaborate the template a bit more so it doesn't add that white space